src/main/java/com/revature/dao/EmployeeDao.java

```diff
@@ -1,40 +1,70 @@
 package com.revature.dao;
 
+import java.util.ArrayList;
 import java.util.List;
 
-import com.revature.models.employee.Employee;
-import com.revature.models.employee.EmployeeRole;
+
+
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.query.Query;
+
+import com.revature.models.Employee;
+import com.revature.models.EmployeeRole;
+import com.revature.utils.SessionHelper;
 
 public class EmployeeDao implements IDao<Employee>{
 
   @Override
   public List<Employee> selectAll() {
-    // TODO Auto-generated method stub
-    return null;
+    List<Employee> employees = new ArrayList<Employee>();
+    try {
+      Session session = SessionHelper.getSession();
+      Query query = session.createQuery("FROM Employee");
+      employees = query.list();
+    } catch (HibernateException e) {
+      e.printStackTrace();
+    }
+    return employees;
   }
 
   @Override
   public Employee selectById(int id) {
-    // TODO Auto-generated method stub
+    try {
+      Session session = SessionHelper.getSession();
+      session.beginTransaction();
+      Employee entity = (Employee) session.get(Employee.class, id);
+      session.getTransaction().commit();
+      return entity;
+    } catch (HibernateException e) {
+      e.printStackTrace();
+    }
     return null;
   }
 
-  @Override
-  public Employee update(Employee entity) {
-    // TODO Auto-generated method stub
-    return null;
-  }
-
-  @Override
-  public Employee delete(Employee entity) {
-    // TODO Auto-generated method stub
-    return null;
-  }
   
   public static void main(String[] args) {
+    List<Employee> employees;
     EmployeeDao employeeDao = new EmployeeDao();
     Employee employee = new Employee("James", "May", "Top Gear", "Slow", "Email", EmployeeRole.EMPLOYEE);
+    employeeDao.insert(employee);
+    
+    employees = employeeDao.selectAll();
+    System.out.println(employees);
+
+    System.out.println("\n\n\n");
+    System.out.println("UPDATING");
+    employee = employees.get(0);
+    employee.setFirstName("John");
+
+    employees = employeeDao.selectAll();
+    System.out.println(employees);
 
-    System.out.println(employeeDao.insert(employee));
+    System.out.println("\n\n\n");
+    System.out.println("DELETING");
+    employee = employees.get(0);
+    employeeDao.delete(employee);
+    employees = employeeDao.selectAll();
+    System.out.println(employees);
   }
```

Comment:
I think it's best to move this to a test class so that the WAR package file only has one `main` method which is for the app itself